src/config/configStore.ts

```diff
@@ -69,7 +69,7 @@ export abstract class BaseConfigStore<T extends BaseConfigStore.Options> extends
   public constructor(options: T) {
     super(options);
     this.options = options;
-    this.setContents(this.options.contents || {});
```

Comment:
How come this isn't needed anymore? Done by `setContents`?

Reply:
Yeah, `setContents` does this so it's not needed.